tests/phpunit/CRM/SMS/BAO/ProviderTest.php

```diff
@@ -64,12 +64,12 @@ public function testCreateAndUpdateProvider() {
       'is_active' => 1,
       'api_type' => 1,
     );
-    CRM_SMS_BAO_Provider::saveRecord($values);
+    CRM_SMS_BAO_Provider::create($values, array());
```

Comment:
I tend to think it only matters to test the api - the BAO is tested in the process & the api is the recommended way to call it. BAO can change, but api is the one we commit to